wrapanapi/systems/lenovo.py

```diff
@@ -112,6 +122,32 @@ def list_servers(self):
         self._servers_list = inventory
         return inventory
 
+    def list_switches(self):
+        if not self._switch_list:
+            instance = self._service_instance(LenovoSwitch.API_PATH)
+            raw_switches = instance.get("switchList", None)
+            if raw_switches:
+                for switch_data in raw_switches:
+                    switch = LenovoSwitch(switch_data, system=self)
+                    self._switch_list.append(switch)
+        return self._switch_list
```

Comment:
I still don't believe its a good idea to store the switches in this way, as the switches on the system could change, and this list would be static.